docs/tasks/run-application/configure-pdb.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Specifying a Disruption Budget for your Application
+---
+
+{% capture overview %}
+
+This page shows how to limit the number of concurrent disruptions
+that your application experiences, allowing for higher availability
+while permitting the cluster administrator to manage the clusters
+nodes.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+* You are the owner of an application running on a Kubernetes cluster that requires
+  high availability.
+* You should know how to deploy [Replicated Stateless Applications](/docs/tasks/run-application/run-stateless-application-deployment.md)
+  and/or [Replicated Stateful Applications](/docs/tasks/run-application/run-replicated-stateful-application.md).
+* You should have read about the [Pod Disruption Budget concept](/docs/tasks/run-application/configure-pdb.md).
```

Comment:
Changed to [Configure a Pod Disruption Budget](https://kubernetes.io/docs/tasks/administer-cluster/configure-pod-disruption-budget/).